src/NServiceBus.Core/Serialization/SerializationFeature.cs

```diff
@@ -8,17 +9,15 @@
     using MessageInterfaces.MessageMapper.Reflection;
     using Pipeline;
     using Serialization;
+    using Settings;
     using Unicast.Messages;
 
     class SerializationFeature : Feature
     {
         public SerializationFeature()
         {
             EnableByDefault();
-            Defaults(s =>
-            {
-                s.SetDefault("AdditionalDeserializers", new List<SerializationDefinition>());
-            });
+            Defaults(s => { s.ApplySerializationDefaults(); });
```

Comment:
since we have a "get or create" logic in `GetAdditionalSerializers` I don't think we need to configure a default?